go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/devfile/library
 go 1.13
 
 require (
-	github.com/devfile/api/v2 v2.1.0
+	github.com/devfile/api/v2 v2.0.0-20210527100059-8230c0835f1d
```

Comment:
interestingly this is going back down one level? 🤔

Reply:
It's the version pulled from main branch. I would expect it's `2.1.0-<timestamp>` as well. 
Need to take a deeper look into api repo. But it shouldn't affect anything in library repo.

Reply:
Any updates on this before I approve the PR? why its 2.0.0..

Reply:
I asked John. He has no idea on why this version has not been updated to 2.1.0. but he said won't affect anything. 
Also this should not be a common issue with importing devfile/api, not a library issue.